public-event-list.md

```diff
@@ -136,268 +138,566 @@ To get the custom included in the Public Event List payload, first create a Fiel
-          },
-          "detail": null,
-          "allocations": [
-            {
-              "id": "a078000000Gh8mCAAR",
-              "name": "Side",
-              "instanceId": "a068000000OjDHlAAN",
-              "sortOrder": 0,
-              "soldOut": false,
-              "levels": [
-                {
-                  "id": "a048000000IuI3kAAF",
-                  "name": "Fri/Sat/Sun",
-                  "allocationId": "a078000000Gh8mCAAR",
-                  "sortOrder": null,
-                  "price": 39.5,
-                  "fee": 4
-                }
-              ]
-            },
-            {
-              "id": "a078000000Gh8mDAAR",
-              "name": "Center",
-              "instanceId": "a068000000OjDHlAAN",
-              "sortOrder": 1,
-              "soldOut": false,
-              "levels": [
-                {
-                  "id": "a048000000IuI3lAAF",
-                  "name": "Fri/Sat/Sun",
-                  "allocationId": "a078000000Gh8mDAAR",
-                  "sortOrder": null,
-                  "price": 42,
-                  "fee": 4
-                }
-              ]
-            },
-            {
-              "id": "a078000000Gh8mFAAR",
-              "name": "Accessible Side",
-              "instanceId": "a068000000OjDHlAAN",
-              "sortOrder": 3,
-              "soldOut": false,
-              "levels": [
-                {
-                  "id": "a048000000IuI3nAAF",
-                  "name": "Fri/Sat/Sun",
-                  "allocationId": "a078000000Gh8mFAAR",
-                  "sortOrder": null,
-                  "price": 39.5,
-                  "fee": 4
-                }
-              ]
-            },
-            {
-              "id": "a078000000Gh8oIAAR",
-              "name": "Accessible",
-              "instanceId": "a068000000OjDHlAAN",
-              "sortOrder": 4,
-              "soldOut": false,
-              "levels": [
-                {
-                  "id": "a048000000IuI3oAAF",
-                  "name": "Fri/Sat/Sun",
-                  "allocationId": "a078000000Gh8oIAAR",
-                  "sortOrder": null,
-                  "price": 42,
-                  "fee": 4
-                }
-              ]
-            }
-          ]
-        }
-      ]
+  "events" : [ {
+    "type" : "Tickets",
+    "sortOrder" : 1,
+    "smallImagePath" : null,
+    "purchaseUrl" : "https://sandbox-littledipper-aries-5925-1786f2f1715.cs45.force.com/ticket/#/events/a178A000002gtOIQAY",
+    "name" : "Test Event",
+    "largeImagePath" : null,
+    "instances" : [ {
+      "venueId" : "a1A8A000001ZPAUUA4",
+      "soldOut" : false,
+      "seatingType" : "Pick Your Own Seats",
+      "saleStatus" : "On Sale",
+      "purchaseUrl" : "https://sandbox-littledipper-aries-5925-1786f2f1715.cs45.force.com/ticket/#/instances/a0W8A000002udGOUAY",
+      "noSaleMessage" : null,
+      "name" : "Access Code Event (PYOS) Dec. 25th 2029 8pm",
```

Comment:
Let's get rid of these fake looking names; in the real world, it would just be the "Dec. 25th, 2029 8pm"